x-pack/plugins/security/server/authorization/reset_session_page.test.tsx

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { renderToStaticMarkup } from 'react-dom/server';
+import { ResetSessionPage } from './reset_session_page';
+
+describe('ResetSessionPage', () => {
```

Comment:
I'm normally not a fan of using external snapshots, but it feels appropriate for this component. My only concern is that if the Fonts component changes, this snapshot breaks. I don't know how often that may occur, but we should probably mock it to be safe:

```suggestion
jest.mock('../../../../../src/core/server/rendering/views/fonts', () => ({
  Fonts: () => <>MockedFonts</>,
}));

describe('ResetSessionPage', () => {
```